recipes/llvm-meta/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: llvm-meta
+  version: 5.0.0
+
+build:
+  noarch: generic
+  number: 0
+
+test:
+  commands:
+    - echo 'Meta package for tracking llvm version'
```

Comment:
Could do something like [this]( https://github.com/conda-forge/conda-env-feedstock/blob/master/recipe/run_test.py ) if you want a better test. Not crucial though.

Reply:
I don't think it's worth it